proposed/event-dispatcher-meta.md

```diff
@@ -199,12 +199,18 @@ Matthew Weier O'Phinney
 
 ## 6. Votes
 
-* [Entrance vote](https://groups.google.com/d/topic/php-fig/6kQFX-lhuk4/discussion)
+* [Entrance vote][]
```

Comment:
We should use exactly one style for links everywhere; we had a mix of different styles. I chose for the "annotated" variety, as it's more consistent with other specifications.